crates/stackable-operator/src/commons/listener.rs

```diff
@@ -68,6 +68,34 @@ pub enum ServiceType {
     ClusterIP,
 }
 
+impl ServiceType {
+    pub fn to_kubernetes_literal(&self) -> String {
+        match self {
+            ServiceType::NodePort => "NodePort".to_owned(),
+            ServiceType::LoadBalancer => "LoadBalancer".to_owned(),
+            ServiceType::ClusterIP => "ClusterIP".to_owned(),
+        }
+    }
+}
```

Comment:
I think we can probably just impl `std::string::ToString`? Or is there something special about the Kubernetes literal (if there was, I'd assume we'd be implementing a trait anyway)?

Reply:
1e8571c